library/core/src/ops/unsize.rs

```diff
@@ -112,6 +112,9 @@ impl<T: ?Sized + Unsize<U>, U: ?Sized> CoerceUnsized<*const U> for *const T {}
 ///     T: Unsize<U>,
 /// {}
 /// ```
+///
+/// [^1]: Formerly known as *object safety*.
+/// [^2]: Formerly known as *object safe*.
```

Comment:
awkward duplication

Reply:
It seems reasonable to remove second footnote given that dyn-incompatiblity was previously defined.

Reply:
Done